# "My Introduction"
> "Posting after a long time"

- toc: false
- branch: master
- badges: true
- comments: true
- categories: [test, jupyter]
- image: images/test_images/black-and-white-city-houses-skyline.jpg
- hide: false
- search_exclude: false
- metadata_key1: metadata_value1
- metadata_key2: metadata_value2
- sticky_rank: 1

# Hey there!

I am actually blogging, and this is my first message to the world.

In [1]:
#hide
import numpy as np
import pandas as pd

In [2]:
#hide_input
df = pd.DataFrame.from_dict({'x': np.random.randint(0,100,100), 'y': np.random.randint(0,100,100)})
df.head()

x   y
0  42  10
1  64  58
2  68  49
3   3  54
4  88  63

In [3]:
#collapse

df = pd.DataFrame.from_dict({'x': np.random.randint(0,100,100), 'y': np.random.randint(0,100,100)})
df.head()

x   y
0  58  41
1   0  78
2  33  11
3   2  17
4  84  56

In [23]:
#collapse_show

df = pd.DataFrame.from_dict({'x': np.random.randint(0,100,100), 'y': np.random.randint(0,100,100)})
df.head()

x   y
0  10  15
1  29  56
2  78  23
3  91  15
4  64   2

In [24]:
#collapse_output

df = pd.DataFrame.from_dict({'x': np.random.randint(0,100,100), 'y': np.random.randint(0,100,100)})
df.head()

x   y
0  93  24
1  54  61
2  29  80
3  92  40
4  88  60

> youtube: https://youtu.be/XfoYk_Z5AkI

We can also post images to the blog.

![](https://www.fast.ai/images/fastai_paper/show_batch.png "Credit: You know what you need to credit this for")

## GitHub Flavored Emojis

Typing `I give this post two :+1:!` will render this:

I give this post two :+1:!

> twitter: https://twitter.com/some-link

fastpages builds upon the minima theme. If you want to customize the styling or layout of fastpages, you can find instructions in minima's README. It is a good idea to read the full contents of the README to understand the directory structure. Furthermore, it is a good idea to have a basic understanding of Jekyll before customizing your theme. For those new to Jekyll, the official docs are a good place to start. Concretely, you can override css in fastpages in **_sass/minima/custom-styles.scss**. NOTE that minima's "skins" feature is currently incompatible with fastpages' css settings.

This adds a linked superscript {% fn 15 %}

{{ "This is the actual footnote" | fndetail: 15 }}

This adds a linked superscript {% fn 20 %}

{{ 'This is the actual footnote with a [link](www.github.com) as well!'  | fndetail: 20 }}

> Warning: There will be no second warning!

> Important: Pay attention! It's important

> Tip: This is my tip. will render this:

> Note: Take note of this. will render this:

> Note: A doc link to [an example website: fast.ai](https://www.fast.ai/) should also work fine. will render in the docs: 